src/client/pydaos/torch/torch_shim.c

```diff
@@ -86,14 +86,15 @@ static PyObject *
 __shim_handle__module_fini(PyObject *self, PyObject *args)
 {
 	int rc = daos_fini();
+	/* This is only for NLT tests when agent stops before python unloads the module.
+	   Should be removed once NLT tests replaced by functional tests
+	 */
+	if (rc == -DER_UNINIT) {
+		rc = 0;
+	}
```

Comment:
this looks like a hack to me. it looks like the test needs to be fixed vs adding this hack to acomodate the test.

Reply:
agreed, removing this